lib/http-bridge/test/integration/Main.hs

```diff
@@ -124,9 +128,11 @@ main = hspec $ do
         wait "cardano-node-simple" (waitForCluster nodeApiAddress)
         wait "cardano-http-bridge" (threadDelay oneSecond)
         nl <- HttpBridge.newNetworkLayer bridgePort
-        cardanoWalletServer nl 1337
+        cardanoWalletServer nl portM
         wait "cardano-wallet" (threadDelay oneSecond)
-        let baseURL = "http://localhost:1337/"
+        let baseURL = case portM of
+                Just port -> "http://localhost:" <> T.pack (show port) <> "/"
+                Nothing -> "http://localhost:9080/"
```

Comment:
Perhaps this can be:
```hs
 "http://localhost:" <> show defaultPort <> "/"
```
I'm assuming that your intention is for this `9080` to be the **same value** as the one used in `Wallet.Api.Server`? (Feel free to correct my assumption.)

Reply:
Fixed, the server now returns the port it has used.